libraries/DataFlash/DataFlash_MAVLinkLogTransfer.cpp

```diff
@@ -66,6 +66,11 @@ void DataFlash_Class::handle_log_message(GCS_MAVLINK &link, mavlink_message_t *m
  */
 void DataFlash_Class::handle_log_request_list(GCS_MAVLINK &link, mavlink_message_t *msg)
 {
+    if (_log_sending_chan >= 0) {
+        link.send_text(MAV_SEVERITY_INFO, "Download in progress");
```

Comment:
Humm, message is better but it still doesn't say what it is about. To make it completely clear I think the message should be "*Log download in progress*"